app/components/signup-form.js

```diff
@@ -67,20 +53,18 @@ export default Component.extend({
     }
   },
 
-  _submit() {
+  _submit: task(function* () {
     let credentials = {
       identification: this.get('user.email'),
       password: this.get('user.password')
     };
 
-    this.get('user').save().then(() => {
-      this._signIn(credentials);
+    let promise = this.get('user').save().then(() => {
+      this.get('signIn')(credentials);
     }).catch((error) => {
-      let payloadContainsValidationErrors = error.errors.some((error) => error.status === 422);
-
-      if (!payloadContainsValidationErrors) {
-        this.controllerFor('signup').set('error', error);
-      }
+      this.get('handleErrors')(error);
     });
-  }
+
+    yield promise;
+  }).drop()
```

Comment:
@sbatson5 I'm a little confused by this. Does this mean that a request will still continue on to the server?